root/components/Relationships.js

```diff
@@ -91,18 +94,40 @@ const Relationships = ({
     ? {names: [{artist: source, joinPhrase: '', name: source.name}]}
     : (source.artistCredit || null);
 
+  const heading = noRelationshipsHeading
+    ? null
+    : <h2 className="relationships">{l('Relationships')}</h2>;
+
   return (
     <>
       {relationships.length ? (
         <>
-          {noRelationshipsHeading ? null : (
-            <h2 className="relationships">{l('Relationships')}</h2>
-          )}
+          {heading}
           <StaticRelationshipsDisplay
             hiddenArtistCredit={hiddenArtistCredit}
             relationships={relationships}
           />
         </>
+      ) : srcRels?.length ? (
+        <>
+          {heading}
+          <p>
+            {exp.l(
+              '{link} only has relationships that are displayed elsewhere.',
+              {link: <EntityLink entity={source} />},
+            )}
```

Comment:
I do wish we could make this a bit less vague by saying where "elsewhere" is. As @yvanzo suggested in the meeting notes:

> Maybe we should just leave a note about event and work relationships to the user when the relationships tab is empty.

Having a custom string for each entity type doesn't seem ideal, but maybe a list of links to the other tabs would work? I'm not sure what's best here. Do you think this string is the best we can do until we can make more substantive UI improvements? If so I'm okay with merging it as-is.

Reply:
I wasn't sure how to do this otherwise. I guess we could check every time whether one or more event rels exist. Work rels are actually displayed, just under the appearances table further below, same as recording rels, for example (which are also filtered out).

Reply:
Maybe separate strings wouldn't be so bad then. Since artists and labels are the only entities with dedicated relationship tabs, the only two places you'd have to tell the user to check is (1) the events tabs for artist or (2) the index page for labels, right?

Reply:
Hmm, actually I'm not even sure whether there's currently any rel for labels that does not appear on the relationships tab, either as in the relationship list or the table. So we probably only need this for artist, heh.